src/adapters/icon_manager.js

```diff
@@ -51,13 +51,17 @@ export default class IconManager {
   }
 }
 
-export function createIcon(iconOptions, hoverEffect = false) {
+export function createIcon(iconOptions, name, hoverEffect = false) {
   const icon = IconManager.get(iconOptions)
 
+  const markerLabel = name ? `<div class="marker-label">${name}</div>` : ''
   const element = document.createElement('div')
   element.innerHTML = `
-    <div class="marker-container${hoverEffect ? ' poi-hoverable' : ''}">
-      <i class="icon icon-${icon.iconClass}"></i>
+    <div class="marker">
+      <div class="marker-container${hoverEffect ? ' poi-hoverable' : ''}">
```

Comment:
The string template can be simplified by something like : ``${hoverEffect && ' poi-hoverable'}``

Reply:
Unfortunaly we're not in jsx, so it doesn't work 😬 if we do this, when `hoverEffect` is `true` it work as expected, but if `false`, we have `false` instead of `''` in the value.
```
`marker-container${true && ' poi-hoverable'}` // "marker-container poi-hoverable"
`marker-container${false && ' poi-hoverable'}` // "marker-containerfalse"
```